src/forms/tests/__snapshots__/fieldGroup.test.jsx.snap

```diff
@@ -1,743 +1,15 @@
-                          componentClass="div"
-                        >
-                          <div
-                            className=""
-                          >
-                            <Radio
-                              bsClass="radio"
-                              checked={false}
-                              disabled={false}
-                              id="selectOne2"
-                              inline={false}
-                              name="radioGroup"
-                              onBlur={[Function]}
-                              onChange={[Function]}
-                              onFocus={[Function]}
-                              title=""
-                              value="Second radio"
-                            >
-                              <div
-                                className="radio"
-                              >
-                                <label
-                                  title=""
-                                >
-                                  <input
-                                    checked={false}
-                                    disabled={false}
-                                    id="selectOne2"
-                                    name="radioGroup"
-                                    onBlur={[Function]}
-                                    onChange={[Function]}
-                                    onFocus={[Function]}
-                                    type="radio"
-                                    value="Second radio"
-                                  />
-                                  Radio button 2
-                                </label>
-                              </div>
-                            </Radio>
-                          </div>
-                        </Col>
-                      </div>
-                    </FormGroup>
-                  </FinalFormComponent>
-                </FinalFormRadio>
-              </Field>
-            </div>
-          </Col>
-          <Col
-            bsClass="col"
-            className="has-error"
-            componentClass="div"
-            xs={10}
-            xsOffset={2}
-          >
-            <div
-              className="has-error col-xs-10 col-xs-offset-2"
-            >
-              <ErrorBlock
-                name="some group"
-                validateOnMount={false}
-              >
-                <Field
-                  format={[Function]}
-                  name="some group"
-                  parse={[Function]}
-                  render={[Function]}
-                  subscription={
-                    Object {
-                      "error": true,
-                      "touched": true,
-                    }
-                  }
-                />
-              </ErrorBlock>
-            </div>
-          </Col>
-        </div>
-      </Col>
-    </FieldGroup>
-  </ReactFinalForm>
-</Group>
+<ReactFinalForm
```

Comment:
So this snapshot test does not really tells us anything. I would try use:
```javascript
... = shallow(<Component />).dive()
```
It should render to the `<FieldGroup />` component because that is what we actually want to test it. This is a common problem with components that require some context (Store for redux, Router for Links etc.)

more info about dive: https://airbnb.io/enzyme/docs/api/ShallowWrapper/dive.html

Now if that does not help we can just get rid of snapshot test completely, because there is no point snapshoting wrapper component that only provides context.